tsconfig.server.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "dist",
```

Comment:
Do we really need "outDir", "noEmit", "isolatedModules", and "include" in this config?

Can't the config be simplified to 

```
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "module": "commonjs",
    "target": "es2017"
  }
}
```
?